tesseract-core/src/lib.rs

```diff
@@ -655,12 +655,24 @@ impl Schema {
         let sort = if let Some(ref s) = query.sort {
             // sort column needs to be named by alias
             // Checking if we are going to sort by RCA or not
-            let measure_name = &s.measure.to_string();
-            if measure_name.contains("RCA") {
-                Some(SortSql {
-                    direction: s.direction.clone(),
-                    column: format!("rca"),
-                })
+            let measure_name = s.measure.to_string();
+            if let Some(ref rca) = query.rca {
+                if measure_name == format!("{} RCA", rca.mea.clone().to_string()){
```

Comment:
do we need `clone()` & `to_string()` here? can we just use `to_string()`?

Reply:
Yep we don't need it I added by mistake